content/chainguard/libraries/java/build-configuration.md

```diff
@@ -371,6 +371,34 @@ repositories {
 }
 ```
 
+The following listing shows a valid `init.gradle` file. It wraps the
+`repositories` element with `allprojects` so that the scope of the file affects
+all projects built locally with Gradle. It also allows for `buildscript`
```

Comment:
The sentence "It also allows for `buildscript` download from the remote URL." doesn't read correctly to me, like it should be one of the following:

"It also allows for `buildscript` to be downloaded from the remote URL."

or 

"It also allows for `buildscript` downloads from the remote URL."

I'm not clear on which of those are correct, or even if the original phrasing is correct.

Reply:
Reworded to make it more explicit.